node/rate_limiter.js

```diff
@@ -72,7 +75,18 @@ function RateLimiter(options) {
     }
     var self = this;
 
+    // stats
+    self.statEmitter = options.statEmitter || new StatEmitter();
+    self.rateLimiterEmptyTags = new stat.RateLimiterEmptyTags();
+    self.serviceRpsStat = self.statEmitter.defineGauge('tchannel.rate-limiting.service-rps');
```

Comment:
yes, I used a wrong sample rate. I will make it every one second. Then the cost should be low, since it will be just 4 stats per second per process.